addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -14,9 +14,8 @@
 
         <template id="l10n_ch_swissqr_template">
             <t t-set="o" t-value="o.with_context(lang=lang)"/>
-            <t t-call="web.external_layout">
-                <!-- TODO master: remove this -->
-                <script t-if="0">document.body.className += " l10n_ch_qr";</script>
+            <t t-set="company" t-value="o.company_id"/>
+            <t t-call="web.external_layout_standard">
```

Comment:
I'm not sure about this line. It's one of the layout you can choose, right? Why this one and not the others then?